bot/exts/info/pypi.py

```diff
@@ -17,6 +17,7 @@
 PYPI_COLOURS = itertools.cycle((Colours.yellow, Colours.blue, Colours.white))
 
 ILLEGAL_CHARACTERS = re.compile(r"[^a-zA-Z0-9-.]+")
+NOT_FOUND_DELETE_DELAY = RedirectOutput.delete_delay
```

Comment:
I think with multiple renamed uses like this it'd be better to move it value to a more appropriate key, although I'm not sure where something general like it would fit

Reply:
I'm not sure either, but I think this should be discussed within the future. 

I'm going to rename `NOT_FOUND_DELETE_DELAY` to `ON_ERROR_DELETE_DELAY` to describe the situations more accurately.